nrf_security/Kconfig

```diff
@@ -0,0 +1,477 @@
+menu "Nordic Security"
+
+config NORDIC_SECURITY_BACKEND
+	bool
+	prompt "Use Nordic provided security backend"
+	depends on SOC_FAMILY_NRF
+	select NRFXLIB_CRYPTO
+	help
+	  Help message
```

Comment:
Tell me more :).

Remember to point to documentation if options will not be explained fully in the help text. Users are very likely to happen upon this while exploring Menuconfig

Reply:
Classified :p 

Will add some more text.